.gitignore

```diff
@@ -8,6 +8,9 @@ appPackage/build
 # python virtual environment
 .venv/
 src/__pycache__/
+src/utils/__pycache__/
```

Comment:
minor: is there a way to ignore folder named `__pycache__` in the whole repo? So we don't need to repeat it for every folder.